modules/999-helm/monitoring/prometheus-rules/deprecated-versions.yaml

```diff
@@ -1,51 +1,34 @@
 - name: d8.helm-releases-resource-versions
   rules:
-  - alert: HelmReleaseHasResourcesWithDeprecatedVersions
+  - alert: HelmReleasesHasResourcesWithDeprecatedVersions
     expr: |
-      max by (helm_release_name) ((max without(instance, pod, hook, job) (resource_versions_compatibility) == 1)
-      * on (module) group_left()(count by(module)(max without(instance, pod, hook, job) (resource_versions_compatibility) == 1) < 50) ^ 0)
+      (count by (k8s_version) (max by (helm_release_namespace, helm_release_name, resource_namespace, resource_name, api_version, kind, k8s_version) (resource_versions_compatibility) == 1)) > 0
     labels:
       tier: cluster
       severity_level: "9"
     annotations:
       plk_markup_format: markdown
       plk_protocol_version: "1"
       plk_pending_until_firing_for: "10m"
```

Comment:
plk_pending_until_firing_for: "10m" -> for: "10m"

Reply:
\+